packages/suite/src/constants/suite/coinjoin.ts

```diff
@@ -8,3 +8,13 @@ export const COINJOIN_PHASE_MESSAGES: Record<RoundPhase, TranslationKey> = {
     [RoundPhase.TransactionSigning]: 'TR_COINJOIN_PHASE_3_MESSAGE',
     [RoundPhase.Ended]: 'TR_COINJOIN_PHASE_4_MESSAGE',
 };
+
+/**
+ * Values are upper limits of anonymity level for each status.
+ */
+export enum AnonymityStatus {
```

Comment:
`AnonymityLevelBoundry` is clearer imo, even without a comment. Also, we are using `level` not `status` everywhere 🙏

Reply:
Yeah, I originally named it `AnonymityLevelBoundary` but then I realized, that it's not used as a boundary, but as status e.g. [`anonymityStatus === AnonymityStatus.Bad`]( https://github.com/trezor/trezor-suite/pull/6569/files#diff-bb67d26a6499c031810d7e10f338973ec63c1f52b9e578e8b41f6a265f6cf727R119). And it's different from the anonymity level (number between 1 and 100), status is Bad, Medium, Good or Great